Framework/DataObjects/test/PeakTest.h

```diff
@@ -123,6 +123,12 @@ class PeakTest : public CxxTest::TestSuite {
     check_Contributing_Detectors(p2, expectedIDs);
   }
 
+  void test_ConstructorFromLeanElasticPeak() {
+    const LeanElasticPeak &lpeak = LeanElasticPeak(V3D(1, 2, 3));
```

Comment:
Can you have an example using a goniometer that is not identity. Then actually check that the `.getQLabFrame` and `.getQSampleFrame` give the same results between Peak and LeanElasticPeak.

Reply:
More robust unit test following the example in LeanElasticPeakTest has been implemented.